sdcm/nemesis/monkey/views_indexes.py

```
@@ -0,0 +1,277 @@
+        ) as working_node:
+            ks_name, base_table_name = self.runner.random.choice(ks_cfs).split(".")
+            view_name = f"{base_table_name}_view_{str(uuid4())[:8]}"
+            with self.runner.cluster.cql_connection_patient(node=working_node, connect_timeout=600) as session:
+                try:
+                    create_materialized_view_for_random_column(session, ks_name, base_table_name, view_name)
+                    wait_materialized_view_building_tasks_started(session, ks_name, view_name)
+                except ViewFinishedBuildingException:
+                    drop_materialized_view(session, ks_name, view_name)
+                    raise UnsupportedNemesis(
+                        f"Skip nemesis because view {ks_name}.{view_name} has already finished building"
+                    )
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)